### PR TITLE
trivial unit test for both histogrammar and matplotlib

### PR DESCRIPTION
**Pull request by davidlange6**

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: PhysicsTools/PythonAnalysis/test/BuildFile.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<environment>
```
Comment (smuzaffar) [inline PhysicsTools/PythonAnalysis/test/BuildFile.xml]: @davidlange6 , there is no need to have <environment> </environment> tag. 
If you want to run these tests whenever we update matplotlib or histogrammar then please add dependency on py2-matplotlib and histogrammar.

**Comment by davidlange6** [comment]
done (hopefully)

**Comment by smuzaffar** [comment]
please test

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
please test

**Pull request merged**